src/cirrus/lambda_functions/api.py

```diff
@@ -106,7 +106,11 @@ def _results_transform(
             "period": ts,
             "interval": interval,
             "states": [
-                {"state": state, "unique_count": state_val[0], "count": state_val[1]}
+                {
```

Comment:
I believe we'll need a corresponding change to the dashboard to support this new state

Reply:
cirrus-dashboard#8 ☑️